### PR TITLE
Upgrade tube_example to 1.4.3

### PR DESCRIPTION
**Pull request by sorsaffari**
# Why is this PR needed?
to allow the community to experiment with the tube_example with the latest version of Grakn

# What does the PR do?
makes the tube_example compatible with 1.4.3

# Does it break backwards compatibility?
no

# List of future improvements not on this PR
needs to be heavily refactored to:
- provide a better learning experience
- make a more performant use of sessions and transactions (which were not available at the time of the initial writing of this example)
- optimise some of the slow queries

### CODE REVIEW

File: applications/tube-network/tube_network_example/README.md [hunk 1]
```diff
@@ -71,56 +36,17 @@ Check Grakn is up and running: `./grakn server status`
 
 To import, run [timetable_migration.py](src/migrations/timetable_migration.py), either in your IDE, or from the grakn_examples directory as follows:
 ```bash
-grakn_examples $ python -m tube_network_example.src.migrations.timetable_migration
+python3 -m tube_network_example.src.migrations.timetable_migration
 ```
 
-This is a custom-built python script that just iterates over the downloaded json data.   
+This is a custom-built python script that just iterates over the downloaded json data.
```
Comment (lolski) [inline applications/tube-network/tube_network_example/README.md]: JSON

File: applications/tube-network/tube_network_example/README.md [hunk 2]
```diff
@@ -1,68 +1,33 @@
...
 
-Here we demonstrate a way of modelling the London Underground Network using data acquired from the Transport for London (TFL) website. 
+Here we demonstrate a way of modelling the London Underground Network using data acquired from the Transport for London (TFL) website.
 
 Most of the code given here is written in Python to give an example of how to use the Grakn Python client.
 
 See the quickstart for how to get going immediately, or read on for more info.
 
+## Prerequisites
+- Grakn >= 1.4.2. Learn more about [installing and running Grakn](http://dev.grakn.ai/docs/running-grakn/install-and-run).
```
Comment (lolski) [inline applications/tube-network/tube_network_example/README.md:10]: 1.4.3

**Review by lolski — CHANGES_REQUESTED** [review]
Small nitpick

File: applications/tube-network/tube_network_example/README.md (hunk 2 above)
Comment (sorsaffari) [reply applications/tube-network/tube_network_example/README.md:10]: do you think it should still be >= 1.4.3, even if I've tested and verified that it also works with 1.4.2?

**Review by lolski — APPROVED** [review]

**Pull request merged**